chain/chain/src/flat_storage_creator.rs

```diff
@@ -203,11 +212,13 @@ impl FlatStorageShardCreator {
                 let final_height = final_head.height;
 
                 if final_height > self.start_height {
-                    // If it holds, deltas for all blocks after final head are saved to disk, because they have bigger
-                    // heights than one on which we launched a node. Check that it is true:
+                    // If it holds, deltas for all blocks after final head are saved to disk,
+                    // because they have bigger heights than one on which we
+                    // launched a node. Check that it is true:
                     for height in final_height + 1..=chain_store.head()?.height {
-                        // We skip heights for which there are no blocks, because certain heights can be skipped.
-                        // TODO (#8057): make `get_all_block_hashes_by_height` return empty hashmap instead of error
+                        // We skip heights for which there are no blocks, because certain heights
+                        // can be skipped. TODO (#8057): make
```

Comment:
This isn’t great. Not sure how to best handle it… maybe an empty comment line before the `TODO`?